uint8 movement
open_base/Velocity wheel
open_base/MovementGeneric generic
open_base/MovementBezier bezier
# movement constants
uint8 BEZIER=0
uint8 GENERIC=1
uint8 NONE=2
uint8 WHEEL=3
# type constants
uint8 POSITION_ABSOLUTE=0
uint8 POSITION_RELATIVE=1
uint8 VELOCITY=2
# frame constants
uint8 HYBRID=0
uint8 MOBILE=1
uint8 MOBILE_RAW=2
uint8 WORLD=3
